src/modules/missionHelper/i18n/nb_NO.json

```diff
@@ -0,0 +1,93 @@
+{
+	"diyMission": "Dette oppdraget ser ut til å være en alliansehendelse i stor skala!",
+	"expansions": "Utvikling | Utviklinger",
+	"followups": "Follow-up Mission | Follow-up Missions",
+	"generated_by": "Generert av",
+	"noVehicleRequirements": [
+		"water_needed"
+	],
+	"patients": {
+		"allow_ktw_instead_of_rtw": "En syketransport kan bli brukt.",
+		"helicopter": "Sannsynlighet for Luftambulanse:",
+		"patient_allow_first_responder_chance": "Sannsynlighet for First Responder:",
+		"patient_at_end_of_mission": "NB: Pasienten dukker opp på slutten av oppdraget!",
+		"patient_transport": "Sannsynlighet for transport:",
+		"possible_patient": "Maksimalt antall pasienter",
+		"possible_patient_exact": "kun {n} pasient | kun {n} pasienter",
+		"possible_patient_min": "minimum pasienter",
+		"specializations": "Utdanning/Spesialitet",
+		"title": "Pasienter | {n} pasient | {n} Pasienter"
+	},
+	"prerequisites": {
```

Comment:
![afbeelding](https://user-images.githubusercontent.com/65117490/98439896-40baea80-20f5-11eb-8f1b-5dd1ae1ae70c.png)
Needs only the above catgories